docs/explanation/caching.md

```diff
@@ -38,7 +38,80 @@ _Removing the Remote Cache_:
 If users wish to remove the remote cache without deleting the associated AWS S3 bucket, they can use the following command:
 
 ```
-k8sgpt cache remove --bucket <name>
+k8sgpt cache remove
```

Comment:
users don't neet to specify the resource?

Reply:
Yes, without specifying any resource, it will select the current active cache and convert it into a file-based cache. I have also checked the K8sGPT code and confirmed that there are no other flags present like --bucket on the cache remove command.